custom-payment-flow/server/node-typescript/src/server.ts

```diff
@@ -117,6 +117,23 @@ app.post(
   }
 );
 
+app.get('/payment/next', async (req, res) => {
+  const paymentIntent : any = req.query.payment_intent; 
+  const intent  = await stripe.paymentIntents.retrieve(
+    paymentIntent,
+    {
+      expand: ['payment_method'],
```

Comment:
didn't see expansion in the other languages, do we want to also fetch the related payment method for the other server languages?